doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20.rst

```diff
@@ -0,0 +1,399 @@
+
+Edge Impulse samples
+--------------------
+
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target in the following samples:
+
+  * :ref:`ei_data_forwarder_sample`
+  * :ref:`ei_wrapper_sample`
+
+Matter samples
+--------------
+
+* Updated:
+
+  * All Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
+    It is enabled by default for the release configuration of the following samples:
+
+    * :ref:`matter_lock_sample`
+    * :ref:`matter_light_switch_sample`
+    * :ref:`matter_smoke_co_alarm_sample`
+    * :ref:`matter_window_covering_sample`
+
+  * All Matter samples to enable the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
+
+* Disabled pausing Matter watchdog while CPU is in idle state in all Matter samples.
+  To enable it, set the :ref:`CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP<CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP>` Kconfig option to ``y``.
+
+* :ref:`matter_smoke_co_alarm_sample` sample:
+
+  * Added:
+
+    * Support for ICD dynamic SIT LIT switching (DSLS).
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Updated:
+
+    * The logging format in the standard logging mode to align it with the format used in the multi-domain logging mode.
+    * Support for alternative configurations to use the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>`.
+      The following file suffixes are supported as alternative configurations:
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Bluetooth libraries and services
+--------------------------------
+
+* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+* :ref:`bt_fast_pair_readme` library:
+
+  * Updated the :c:func:`bt_fast_pair_info_cb_register` API to allow registration of multiple callbacks.
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Integrations
+============
+
+This section provides detailed lists of changes by :ref:`integration <integrations>`.
+
+Google Fast Pair integration
+----------------------------
+
+* Added instructions on how to provision the Fast Pair data onto devices without the :ref:`partition_manager` support, specifically for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
+
```

Comment:
MCUboot section is missing. Is it intentional?

Reply:
Yes, MCUboot is not supported on 54h